internal/pkg/service/buffer/store/export.go

```diff
@@ -30,7 +30,7 @@ func (s *Store) CreateExport(ctx context.Context, export model.Export) (err erro
 	_, err = op.MergeToTxn(
 		s.createExportBaseOp(ctx, export.ExportBase),
 		s.createMappingOp(ctx, export.Mapping),
-		s.createTokenOp(ctx, export.ExportKey, export.Token),
+		s.createTokenOp(ctx, model.TokenForExport{ExportKey: export.ExportKey, Token: export.Token}),
```

Comment:
Pridal som do tokenu ExportKey, nech to mame rovnako ako ostatne objekty, bude sa nam to hodit.